### DISCUSSION TITLE
Object composition op(s)

### DISCUSSION THREAD

**Discussion by cristianvogel** [cables-gl/cables_docs#131] [Feature Requests]
I am thinking something along the lines of StringCompose, where inputs can be injected as values into JS or JSON Objects.

At the moment, trying to set up OSC ( see [my post here](https://github.com/cables-gl/cables_docs/discussions/126) ) its difficult to get the most out of the possibilities of Cables as front end for other devices due to the lack of simple Object creation and updating Ops. 

For example, just sending one message over OSC involves quite a bit of wiring?

<img width="752" alt="Screenshot 2022-01-27 at 16 29 59" src="https://user-images.githubusercontent.com/12829050/151390947-e405df78-ff5e-47e2-902c-4ce95131c599.png">

**Comment by steam0r** [comment]
you could actually use StringCompose to construct your json object, then use ParseObject to make it JSON

**Reply by cristianvogel** [reply]
Wouldn't I still need to mess about with uncombining then recombining in order to update a numeric value in the JSON though?
